src/main/java/com/tourism/psk/model/Comment.java

```diff
@@ -0,0 +1,89 @@
+package com.tourism.psk.model;
+
+import javax.persistence.*;
+import java.util.Objects;
+
+@Entity
+public class Comment {
+
+    @Id
+    @GeneratedValue
+    private long id;
+
+    @ManyToOne
+    @JoinColumn
+    private User commentedBy;
+
+    private String text;
+
+    private String date;
```

Comment:
Perhaps use a Date object instead?

Reply:
Would not be compliant with swagger, we can change this later if needed.